operator/pkg/client/roles/client_test.go

```diff
@@ -179,66 +234,3 @@ func TestClient(t *testing.T) {
 		test(t, container)
 	})
 }
-
-// Test utility functions
-func TestCalculateMembershipChanges(t *testing.T) {
```

Comment:
This seems like an import set of tests to have. Is there a reason to not preserve it?